api/interservice/authz/v2/policy_test.go

```diff
@@ -342,28 +342,28 @@ func TestValidationCreatePolicy_Actions(t *testing.T) {
 	classes := map[bool][]string{
 		true: {
 			"*",
-			"infra:ingestNodeRuns:create",
+			"ingest:camelCaseThings:create",
 			"no:camels:here",
 			"*:mark-for-deletion", // legacy
 			"infra:*:create",
-			"infra:ingestNodeRuns:*",
+			"ingest:camelCaseThings:*",
```

Comment:
the "realness" of the action wasn't important here-- these tests are for validation of format.